src/utils/with-react.js

```diff
@@ -167,5 +169,9 @@ export default (createElement, componentClass) => {
   // displayName is important for React testing (e.g. enzyme) and Chrome DevTools plugins
   reactStatelessComponent.displayName = displayName;
 
+  if (version) {
+    defineOnce(finalTagName, class extends componentClass {});
```

Comment:
this needs to be done for native, meaing that native components also must define tag name like so:

`defineOnce('axa-dropdown', class extends componentClass {});`
`defineOnce('axa-dropdown-1-2-3', class extends componentClass {});`

axa-dropdown -> commonjs -> rsv -> axa-button
axa-dropdown -> pod-aem-plib -> axa-button

Reply:
`defineOnce(`${AXADropdown.tagName}-${AXADropdown.version[0]}`, AXADropdown);`